wemake_python_styleguide/visitors/tokenize/statements.py

```diff
@@ -256,3 +258,111 @@ def _post_visit(self) -> None:
             self._check_individual_line(
                 line_tokens, previous_line_token, next_line_token,
             )
+
+
+@final
+class InconsistentComprehensionContext(object):
+    """
+    Context for individual bracket contexts (i.e. within a set of brackets).
+
+    Helper class for InconsistentComprehensionVisitor which stores context
+    for the current (potential) comprehension we are in. Combined with a
+    stack to enable support for nested comprehensions.
+    """
+
+    def __init__(self) -> None:
```

Comment:
Looks like a dataclass / namedtuple to me

Reply:
Converted to dataclass